index.html

```diff
@@ -1,8 +1,95 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Memory exercise</title>
-  </head>
-  <body>
-  </body>
+<head>
+  <title>Memory Excercise</title>
+  <style>
+  div#memory_board{
+  	background:#CCC;
+  	border:#999 1px solid;
+  	width:275px;
+  	height:275px;
+  	padding:24px;
+  	margin:0px auto;
+  }
+  div#memory_board > div{
+  	background: url(tile_bg.jpg) no-repeat;
+  	border:#000 1px solid;
+  	width:71px;
+  	height:71px;
+  	float:left;
+  	margin:10px;
+  	padding:20px;
+  	font-size:64px;
+  	cursor:pointer;
+  	text-align:center;
+  }
+  </style>
+  <script>
+  var memory_array = ['A','A','B','B'];
+  var memory_values = [];
+  var memory_tile_ids = [];
+  var tiles_flipped = 0;
+  Array.prototype.memory_tile_shuffle = function(){
+      var i = this.length, j, temp;
+      while(--i > 0){
+          j = Math.floor(Math.random() * (i+1));
+          temp = this[j];
+          this[j] = this[i];
+          this[i] = temp;
+      }
+  }
+  function newBoard(){
+  	tiles_flipped = 0;
+  	var output = '';
+      memory_array.memory_tile_shuffle();
+  	for(var i = 0; i < memory_array.length; i++){
+  		output += '<div id="tile_'+i+'" onclick="memoryFlipTile(this,\''+memory_array[i]+'\')"></div>';
```

Comment:
In other words, how can we make it [unobtrusive](http://en.wikipedia.org/wiki/Unobtrusive_JavaScript)?